indy_node/server/client_authn.py

```diff
@@ -18,23 +15,10 @@ class LedgerBasedAuthNr(CoreAuthMixin, NaclAuthNr):
     Transaction-based client authenticator.
     """
 
-    write_types = CoreAuthMixin.write_types.union(
-        PoolRequestHandler.write_types
-    ).union(
-        DomainReqHandler.write_types
-    ).union(
-        ConfigReqHandler.write_types
-    )
-
-    query_types = CoreAuthMixin.query_types.union(
-        {GET_TXNS, }
-    ).union(
-        PoolRequestHandler.query_types
-    ).union(
-        DomainReqHandler.query_types
-    ).union(
-        ConfigReqHandler.query_types
-    )
+    write_types = CoreAuthMixin.write_types.union({ATTRIB, SCHEMA, CLAIM_DEF,
```

Comment:
Why do we have to specify it explicitly? Why can't get it from registered request handlers?

Reply:
Because request handlers are registered with node not authenticator.